magpie/setup/magpie-setup-project-zeppelin

```diff
@@ -153,9 +153,18 @@ then
     phoenix_jars="${PHOENIX_HOME}/phoenix-${PHOENIX_VERSION}-server.jar,$phoenix_jars"
     phoenix_jars="${PHOENIX_HOME}/phoenix-spark-${PHOENIX_VERSION}.jar,$phoenix_jars"
     phoenix_jars="${PHOENIX_HOME}/phoenix-core-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    phoenix_jars="${PHOENIX_HOME}/phoenix-server-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    phoenix_jars="${PHOENIX_HOME}/phoenix-server-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    zeppelin_jars="${zeppelin_jars},${phoenix_jars}"
+    
+    Magpie_vercomp ${PHOENIX_VERSION} 4.8.0
+    vercomp_result=$?
+    if [ "${vercomp_result}" == "0" ] || [ "${vercomp_result}" == "1" ]
+    then
+        phoenix_jars="${PHOENIX_HOME}/phoenix-queryserver-${PHOENIX_VERSION}.jar,$phoenix_jars"
+        phoenix_jars="${PHOENIX_HOME}/phoenix-queryserver-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    else
+        phoenix_jars="${PHOENIX_HOME}/phoenix-server-${PHOENIX_VERSION}.jar,$phoenix_jars"
+        phoenix_jars="${PHOENIX_HOME}/phoenix-server-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    fi
+    zeppelin_jars="${zeppelin_jars%,},${phoenix_jars%,}"
```

Comment:
I think the code above this is correct and there's no danger of a dangling comma??